<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerSpectrumDialog</class>
 <widget class="QDialog" name="PowerSpectrumDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Power spectrum</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxConditions">
     <property name="title">
      <string>Conditions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="labelFmin">
            <property name="toolTip">
             <string>Set the lower limit for the frequencies.</string>
            </property>
            <property name="text">
             <string>Min frequency of interest:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxFmin">
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="toolTip">
             <string>Set the lower limit for the frequencies.</string>
            </property>
            <property name="suffix">
             <string>Hz</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelFmax">
            <property name="toolTip">
             <string>Set the upper limit for the frequencies.</string>
            </property>
            <property name="text">
             <string>Max frequency of interest:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBoxFmax">
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="toolTip">
             <string>Set the upper limit for the frequencies.</string>
            </property>
            <property name="suffix">
             <string>Hz</string>
            </property>
            <property name="maximum">
             <number>150</number>
            </property>
            <property name="value">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelChannels">
            <property name="text">
             <string>Channels</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBoxChannels">
            <item>
             <property name="text">
              <string>MEG</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EEG</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_6">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelNfft">
            <property name="toolTip">
             <string>The length of the tapers ie. the windows. The smaller it is the smoother are the PSDs.</string>
            </property>
            <property name="text">
             <string>Length of the tapers (ie. the windows):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxNfft">
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="toolTip">
             <string>The length of the tapers ie. the windows. The smaller it is the smoother are the PSDs.</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>2048</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxLogarithm">
            <property name="toolTip">
             <string>Use logarithmic scale.</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Use logarithmic scale</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_27">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_35">
            <item>
             <widget class="QRadioButton" name="radioButtonSelectLayout">
              <property name="text">
               <string>Select layout</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxLayout"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_36">
              <item>
               <widget class="QRadioButton" name="radioButtonLayoutFromFile">
                <property name="text">
                 <string>Layout from file</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonBrowseLayout">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="labelLayout">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>No layout selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButtonSeriesFromTriggers">
          <property name="text">
           <string>Construct time series from triggers</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddTimeSeries">
          <property name="toolTip">
           <string>Add condition.</string>
          </property>
          <property name="text">
           <string>Add time series</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <widget class="QScrollArea" name="scrollAreaConditions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>255</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>550</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>824</width>
           <height>221</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayoutConditions"/>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxTriggers">
          <property name="text">
           <string>Use triggers for time series</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Start trigger:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxStart">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>End trigger:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxEnd">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxFmin</tabstop>
  <tabstop>spinBoxFmax</tabstop>
  <tabstop>comboBoxChannels</tabstop>
  <tabstop>spinBoxNfft</tabstop>
  <tabstop>checkBoxLogarithm</tabstop>
  <tabstop>radioButtonSelectLayout</tabstop>
  <tabstop>comboBoxLayout</tabstop>
  <tabstop>radioButtonLayoutFromFile</tabstop>
  <tabstop>pushButtonBrowseLayout</tabstop>
  <tabstop>checkBoxTriggers</tabstop>
  <tabstop>comboBoxStart</tabstop>
  <tabstop>comboBoxEnd</tabstop>
  <tabstop>scrollAreaConditions</tabstop>
  <tabstop>pushButtonSeriesFromTriggers</tabstop>
  <tabstop>pushButtonAddTimeSeries</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PowerSpectrumDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PowerSpectrumDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggers</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxStart</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggers</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxEnd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>850</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggers</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggers</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>697</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggers</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonAddTimeSeries</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonLayoutFromFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonBrowseLayout</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonLayoutFromFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelLayout</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
